config/types.go

```diff
@@ -62,6 +62,9 @@ type ServiceInfo struct {
 	// RequestTimeout specifies a timeout (in milliseconds) for
 	// processing REST request calls from other services.
 	RequestTimeout string
+	// EnableNameFieldEscape indicates whether enables NameFieldEscape in this service
+	// The name field escape could allow the system to use special or Chinese characters in the different name fields, including device, profile, and so on.  If the EnableNameFieldEscape is false, some special characters might cause system error.
+	EnableNameFieldEscape bool
```

Comment:
Please add TODO comment to remove in EdgeX 4.0

Reply:
Added comment.